examples/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+if (${WITH_EXAMPLES})
```

Comment:
Also see [option](https://cmake.org/cmake/help/latest/command/option.html) for a way to get this flag into the config. It gives users a checkbox in cmake-gui and a way to provide a tooltip.